src/components/app/context.ts

```diff
@@ -23,7 +23,9 @@ export function initContext(req: any, router: Router, route: Route, config: IApp
   return {
     app: config,
     routePartOf: (name: string) => route.definition.name === name || route.definition.name.startsWith(name),
-    linkTo: (name: string, params: IParameters = {}) => router.findByName(name).composeURL(params),
```

Comment:
I think it would be better to put an extra `absoluteLinkTo(name, params)` function on the context, rather than adding the absolute parameter to `linkTo` - it's not obvious at the call site what the boolean flag does in `linkTo('blah.blah', undefined, true)`, whereas `absoluteLinkTo('blah.blah')` would be clear.